app/grandchallenge/algorithms/models.py

```diff
@@ -222,6 +223,11 @@ def get_absolute_url(self):
     def api_url(self):
         return reverse("api:algorithm-detail", kwargs={"pk": self.pk})
 
+    @property
+    def supports_batch_upload(self):
+        inputs = {inpt.slug for inpt in self.inputs.all()}
+        return inputs == {"generic-medical-image"}
```

Comment:
I'm a bit confused. If this property replaces `use_flexible_inputs`, shouldn't we return `inputs != {"generic-medical-image"}` ?

Reply:
The logic has been reversed, hence the change in the property name. Batch uploads mean that many jobs result from one upload session, which can only happen if the only input to the algorithm is generic.

Reply:
Ah ok, I misread how you're using this property in the template but I see now that indeed it's reversed. Thanks for clarifying!